nightly.html

```diff
@@ -106,7 +109,7 @@ <h3 id="scroll-text" class="hide" style="margin: 2rem 0 0 0;">Scroll horizontall
 </footer>
 
 <script src="dist/js/app.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.0/moment.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.0/moment.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 <script>onNightlyLoad();</script>
```

Comment:
This is a result of the current Handlebars setup, where scripts are injected into the footer via a variable.

In this case, I'd already used the two slots that I'd allocated for scripts, so I just put two in the same variable - hence they ended up on the same line.

This could be improved, I agree. Will require a 3rd slot to allow 3 scripts to be injected via Handlebars.